scripts/standard_configure.sh

```diff
@@ -34,6 +34,7 @@ export KRATOS_APPLICATIONS=
 add_app ${KRATOS_APP_DIR}/LinearSolversApplication
 add_app ${KRATOS_APP_DIR}/StructuralMechanicsApplication
 add_app ${KRATOS_APP_DIR}/FluidDynamicsApplication
+add_app ${KRATOS_APP_DIR}/TopologyOptimizationApplication
```

Comment:
```suggestion
```

such an addition must be discussed in a separate PR, please remove it from this PR